openedx/core/djangoapps/user_api/templates/user_api/edx_ace/deletionnotificationmessage/email/body.txt

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+
+{% blocktrans trimmed %}
+Hello {{full_name}},
+
+We received a deletion request for your account on {{platform_name}}. We're sorry to see you go! 
+
+Your account will be deleted shortly. Account deletion, including removal from email lists, may take a few weeks to fully process through our system. If you want to opt-out of emails before then, please unsubscribe from the footer of any email.
+
+This is an informational email only. If you did not initiate this request, please contact {{contact_email}}.
+
+Best,
+{{platform_name}}
+{% endblocktrans %}
```

Comment:
should each of these paragraphs also have their own blocktrans blocks like the html version?

Reply:
I was going back and forth on this. The reason why I had to do it for the html version was because of the paragraph tags that I didn't want to make translators deal with. This is the text-only version of the e-mail and so it can use simple newlines. I can switch to using the individual `blocktrans` blocks if it makes more sense to do it that way.

Reply:
With individual blocktrans, doesn't that mean the plaintext version can share the same translations as the html version?  If so, then I think it's worthwhile.

Reply:
Sure, they've been added.